core/templates/domain/question/question-backend-api.service.ts

```diff
@@ -181,7 +181,7 @@ export class QuestionBackendApiService {
    * Returns a list of questions based on the list of skill ids and number
    * of questions requested.
    */
-  fetchQuestions(
+ async fetchQuestionsAsync(
```

Comment:
Hm, do you need to make changes in all the places where this method is called? Ditto below.